packages/view/src/Charcoal/View/Twig/TranslatorHelpers.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Charcoal\View\Twig;
+
+use LogicException;
+
+// From 'charcoal-translator'
+use Charcoal\Translator\Translator;
+
+// From 'charcoal-view'
+use Charcoal\View\Twig\HelpersInterface;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * Translating Mustache Templates
+ */
+class TranslatorHelpers extends AbstractExtension
+    implements HelpersInterface
+{
+    /**
+     * Store the translator service.
+     *
+     * @var Translator|null
+     */
+    private $translator;
+
+    /**
+     * Store the given locale (Mustache tag node).
+     *
+     * This must be an available locale on the Translator.
+     *
+     * @var string|null
+     */
+    private $locale;
+
+    /**
+     * Store the given domain for the message (Mustache tag node).
+     *
+     * This must be an available domain on the Translator.
+     *
+     * @var string|null
+     */
+    private $domain;
+
+    /**
+     * @param array $data Class Dependencies.
+     */
+    public function __construct(array $data = null)
+    {
+        if (isset($data['translator'])) {
+            $this->setTranslator($data['translator']);
+        }
+    }
+
+    public function getFilters()
+    {
+        return [
+            new TwigFilter('trans', [$this, 'trans']),
+            new TwigFilter('transchoice', [$this, 'transchoice']),
```

Comment:
Missing whitespace around single-line array declarations.

```suggestion
            new TwigFilter('trans', [ $this, 'trans' ]),
            new TwigFilter('transchoice', [ $this, 'transchoice' ]),
```